docs/reference/search/search-your-data/paginate-search-results.asciidoc

```diff
@@ -81,6 +81,12 @@ NOTE: Search after requests have optimizations that make them faster when the so
 order is `_shard_doc` and total hits are not tracked. If you want to iterate over all documents regardless of the
 order, this is the most efficient option.
 
+IMPORTANT: If the `sort` field is a <<date,`date`>> in some target data streams or indices
+but a <<date_nanos,`date_nanos`>> field in other targets, use the `format`
+parameter to specify a <<mapping-date-format, date format>> for the `sort`
+field. Otherwise, {es} won't interpret the search after parameter correctly in
+each request.
```

Comment:
That works for the format but the result will be incorrect. When mixing date and date_nanos it is important to force the `numeric_type` as explained [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/sort-search-results.html#_sorting_numeric_fields).

Reply:
Good point, I expanded the docs in https://github.com/elastic/elasticsearch/pull/70357/commits/ca1b1cd057d1841ff7bdac4852b5820ebf4e3b10.